x/rewards/keeper/providers.go

```diff
@@ -196,7 +196,14 @@ func (k Keeper) CalculateContributionPercentages(ctx sdk.Context, reward math.In
 
 	// validators_participation = validators_participation_param / (1-community_tax)
 	validatorsParticipationParam := k.GetParams(ctx).ValidatorsSubscriptionParticipation
-	validatorsParticipation = validatorsParticipationParam.Quo((sdk.OneDec().Sub(communityTax)))
+	dividedBy := sdk.OneDec().Sub(communityTax)
```

Comment:
6 lines above I check if communityTax == 1
there is no need for this check in my opinion

Reply:
Correct, removed in 3570cdaae2e28af0354ef0cf46581293ba98cb9b